api/turing/api/appcontext.go

```diff
@@ -81,19 +85,50 @@ func NewAppContext(
 		return nil, errors.Wrapf(err, "Failed initializing cluster controllers")
 	}
 
+	// Initialise Batch components
+	// Since there is only the default environment, we will not create multiple batch runners.
```

Comment:
It would be actually, but I think this is closely related to this comment https://github.com/gojek/turing/pull/54#discussion_r630469028. That we do not want to user to choose the environment to run the ensembler in.